run.js

```diff
@@ -248,15 +249,21 @@ export async function run( {
 			`${ hashValue }-${ run }-${ testUrl }-${ fullBrowser }`
 		);
 
-		const url = buildTestUrl( {
+		const urlOptions = {
 			baseUrl,
 			browserstack,
 			flags,
 			run,
 			jsdom: browser.browser === "jsdom",
 			port,
-			reportId,
 			testUrl
+		};
+
+		const descriptiveUrl = buildTestUrl( urlOptions );
+		const url = buildTestUrl( {
```

Comment:
The `moduleId` replacement and `reportId` are the things left out of the descriptive URL.